dapp/src/utils/networks.ts

```diff
@@ -0,0 +1,51 @@
+// Simple network configuration - just the basics
+export const networks = {
+  testnet: {
+    name: "Testnet",
+    passphrase: "Test SDF Network ; September 2015",
+    rpc: "https://soroban-testnet.stellar.org:443",
+    horizon: "https://horizon-testnet.stellar.org",
+    tansu_contract: "CBCXMB3JKKDOYHMBIBH3IQDPVCLHV4LQPCYA2LPKLLQ6JNJHAYPCUFAN",
+    domain_contract: "CAQWEZNN5X7LFD6PZBQXALVH4LSJW2KGNDMFJBQ3DWHXUVQ2JIZ6AQU6",
```

Comment:
Contract addresses should come from the env. I don't mind seeing the URLs being hard-coded.

For the RPC on mainnet, use https://rpc.lightsail.network